vue/src/vueTextMask.js

```diff
@@ -1,23 +1,91 @@
 import createTextMaskInputElement from '../../core/src/createTextMaskInputElement'
 
 export default {
-  textMaskInputElement: null,
+  template: `
+    <input ref="input"
+      :value="value"
+      @input="updateValue($event.target.value)"
+    />
+  `,
 
-  inputHandler({target: {value}}) {
-    return this.textMaskInputElement.update(value)
+  name: 'masked-input',
+
+  props: {
+    value: {
+      type: String,
+      required: false,
+      default: ''
+    },
+
+    mask: {
+      type: [Array, Function],
+      required: true
```

Comment:
My branch doesn't seem to support providing false for the mask. I am new to git, how can I update it?